packages/cluster/src/common.js

```diff
@@ -80,8 +85,29 @@ function replacer(string, token, value) {
     return string.replace(regexp, value)
 }
 
-function resolveConfiguration(argv = {}, cache = {}, config = {}) {
-    const resolved = Object.assign({}, defaults, config, cache, argv)
+async function resolveConfiguration(argv = {}) {
+    const file = path.join(clusterDir, argv.name, cacheFile)
+
+    let currentCache
+    try {
+        currentCache = JSON.parse(await argv.getCache().read(file))
+    } catch (e) {
+        log.debug('JSON parse of cache file failed', e)
```

Comment:
`log` is undefined here!

Reply:
I noticed this testing the "upgrade" path where an previously-existing cluster is managed (i.e.`d2 cluster down dev --clean`) with the latest cli

Reply:
(went ahead and fixed this myself)

Reply:
Thanks!